packages/amplication-server/src/core/assistant/assistantFunctions.service.ts

```diff
@@ -286,6 +292,27 @@ export class AssistantFunctionsService {
 
       const { version, settings, configurations } = pluginVersion;
 
+      if (
+        configurations &&
+        configurations[REQUIRES_AUTHENTICATION_ENTITY] === "true" &&
+        !authEntityExist
+      ) {
+        const authEntityName = await this.resourceService.getAuthEntityName(
+          serviceId,
+          context.user
+        );
+        if (!isEmpty(authEntityName)) {
```

Comment:
I would write `if (isEmpty())` and add the `authEntityExist = true` at the end after the if (since it's done in both cases and the code would be more readable IMO)

Reply:
thanks, i will pass this time